academy/4-my-own-chain/academy-notes.txt

```diff
@@ -1,5 +1,7 @@
-Student doing Checkers using Ignite v0.21.0 with Cosmos SDK v0.45.4 on Apple M1: https://github.com/msicilia/checkers
+Student doing Checkers using Ignite v0.21.0 with Cosmos SDK v0.45.4 on Apple M1: https://github.com/msicilia/checkers up to query
```

Comment:
"up to query" looks odd. I'm not sure how to interpret it.

Reply:
Ah haha, up to the part of the tutorial that deals about (can-play) queries.